src/plonk/circuit.rs

```diff
@@ -175,19 +175,12 @@ impl TryFrom<Column<Any>> for Column<Instance> {
 /// }
 /// ```
 #[derive(Clone, Copy, Debug)]
```

Comment:
These additional derivations are not logically part of this PR, but they're needed downstream and could be conveniently added here.
```suggestion
#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
```